airbyte-integrations/connectors/source-rki-covid/integration_tests/invalid_config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "incidence_in_days": 100,
+  "deaths_in_days": 100,
+  "recovered_in_days": 100,
+  "frozen_incidence_in_days": 100,
+  "hospitalization_in_days":100
```

Comment:
This is not an example of the connector config.

Reply:
Added corrected connector config.